client/src/components/ui/Modal.svelte

```diff
@@ -0,0 +1,84 @@
+<script lang="ts">
+    import Button from "./Button.svelte";
+
+    export let showModal = false;
+    export let isForm = false;
```

Comment:
This doesn't feel right. Why are we changing the behavior of the modal based on its contents? Shouldn't a component not care about its slotted elements?